src/service/rpc/generic/LoadBalancer.ts

```diff
@@ -42,6 +43,20 @@ interface InitRemoteHostsParams {
   noSSRFCheck?: boolean
 }
 
+enum RequestType {
```

Comment:
don't we have other types? like DELETE for unsubscribe?

Reply:
I dont understand what do you mean - this is different use case. It is only for purpose of logging.

Reply:
nvm, thought it's on a different level